meson.build

```diff
@@ -140,3 +153,13 @@ doxygen = find_program('doxygen', required : get_option('enable_docs') == true)
 if get_option('enable_docs')
   subdir('doc')
 endif
+
+if meson.version().version_compare('>= 0.53')
```

Comment:
The minimum meson version is already this.

Reply:
Thanks for the help. Updated PR should have them addressed.